### DISCUSSION TITLE
How to improve OCR?

### DISCUSSION THREAD

**Discussion by mpek** [SubtitleEdit/subtitleedit#6341] [General]
How can I improve OCR, because obviously some characters are being always recognized wrongly. 
In my test, the language is set to german, because I want to work on german subtitle. 
For example: 
" h " recognized as " b "
" , " recognized as " ' "
Where I can accept the fail with " h " to be a " b ", it is soemwhat strange, that a "low" character appears "high"...
Do you have any recommendations for me?

**Comment by niksedk** [comment]
What OCR method do you use and what is your source?

nOCR is recommend for larger font sizes, while Tesseract/BinaryImageCompare are recommended for smaller fonts.

Use character inspect (double click on lines) to improve matches in nOCR/BinaryImageCompare. Also, it's often a good idea to start with a small % pct.

Read more here: https://www.nikse.dk/subtitleedit/help#importvobsub

**Comment by mpek** [comment]
I see, i was leaving it to default settings (binary image compare) and going to test the other methods. Thank you!
